.github/workflows/build.yml

```diff
@@ -22,7 +22,7 @@ jobs:
       matrix:
         php: [ "8.1", "8.2", "8.3" ]
         symfony: [ "^5.4", "^6.0" ]
-        sylius: [ "~1.13.0" ]
+        sylius: [ "~1.13.0", "~1.14.0" ]
```

Comment:
Please add the same config to `coding_standard.yml` file

Reply:
Done https://github.com/BitBagCommerce/SyliusElasticsearchPlugin/pull/264/files#diff-23bcd199f83782bd46c557da1a512a5e1d2938911bc385c445510b0a71bc3a00R23